src/Microsoft.ML.CodeGen/CodeGenerator/CSharp/CodeGeneratorSettings.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using Microsoft.ML.AutoML;
+
+namespace Microsoft.ML.CodeGenerator.CSharp
+{
+    public class CodeGeneratorSettings
+    {
+        public CodeGeneratorSettings()
+        {
+            // set default value
```

Comment:
Comment style in ML.NET is to begin comment with an uppercase character
```suggestion
            // Set default value
```

Reply:
Thanks for pointing out

Reply:
#resolved